lotus/client.go

```diff
@@ -52,6 +54,16 @@ func New(maddr ma.Multiaddr, authToken string) (*apistruct.FullNodeStruct, func(
 
 }
 
+func NewEmbedded() (*apistruct.FullNodeStruct, func(), error) {
+	util.AvgBlockTime = time.Second * 1
```

Comment:
If we're running in embedded mode, tune some configs since blocks aren't produced at the default _AvgBlockTime=30s_ that Mainnet will run but faster (since is a local devnet)